common/djangoapps/util/milestones_helpers.py

```diff
@@ -408,3 +408,16 @@ def add_user_milestone(user, milestone):
         return None
     from milestones import api as milestones_api
     return milestones_api.add_user_milestone(user, milestone)
+
+
+def get_service():
+    """
+    Returns MilestonesService instance if feature flag enabled;
+    else returns None.
+
+    Note: MilestonesService only has access to the functions
+    explicitly requested in the MilestonesServices class
+    """
+    if not settings.FEATURES.get('MILESTONES_APP', False):
```

Comment:
Yes it did.

However, this code looks robust to not having the feature flag turned on, right? It can still be merged and not break anything while we work to get the flag turned on. That's when we'll need extensive stage testing.